walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimeRange.java

```diff
@@ -49,7 +49,7 @@ public int compare(TimeRange a, TimeRange b) {
   private final int start;
   private final int duration;
 
-  private TimeRange(int start, int duration) {
+  public TimeRange(int start, int duration) {
```

Comment:
Not necessary when using [`fromStartEnd`](https://github.com/semiluna/step2020/blob/49d1ad347eada7a717aff991539595f0af93a032/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimeRange.java#L176).

Changing visibility, esp. for a constructor, should be the last resort. It changes the API and might lead to invalid `TimeRange` objects. In this case, `fromStartEnd` is not the best example as e.g. it does not check if `start` is smaller than `end`, so you could have time-traveler events.

Reply:
You're right, it was lazy of me. Changed it back.